extensions/net/Cargo.toml

```diff
@@ -19,6 +19,7 @@ deno_core = { version = "0.94.0", path = "../../core" }
 log = "0.4.14"
 lazy_static = "1.4.0"
 rustls = "0.19.0"
+rustls-native-certs = "0.5.0"
```

Comment:
Here is the link so you can evaluate this package more closely:
https://github.com/rustls/rustls-native-certs

I'm not affiliated with it at all but I gave it a look over and found it to be good.

It seems to:

* have a compatible open source license
* be under active development
* be very simple, it does only the one thing we would want it to do
* actually work